apps/studio/src/components/Datatable/Datatable.tsx

```diff
@@ -116,7 +116,11 @@ export const Datatable = <T extends object>({
             {rows.length === 0 && emptyPlaceholder}
             {rows.map((row) => {
               return (
-                <Tr key={row.id} borderBottomWidth="1px">
+                <Tr
+                  key={row.id}
+                  borderBottomWidth="1px"
+                  _hover={{ bg: "interaction.muted.main.hover" }}
```

Comment:
nit: maybe `bgColor` instead of `bg`

Reply:
updated in https://github.com/opengovsg/isomer/pull/1083/commits/3777babe6b0a33daeab960563ea9f746b579474e 👍